### PR TITLE
Completed the 3 Features

### PR DESCRIPTION
**Pull request by abeltsew**
Hi Review team,

I have completed the below features and please kindly review my changes.

-  Created a linting setup
- Created an index page
- Updated the Heading style

### CODE REVIEW

**Review by wayungi — COMMENTED** [review]
Hi @abeltsew , 

While you made a great effort in this project 👍🏾 , unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🔴 
- [ ] You have submitted a project with an unprofessional README, Please fill in all sections of the README file. The only optional section is the FAQ

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -1,2 +1,57 @@
...
+
+> The first HTML and CSS project from Microverse
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (AlphaNtihinduka) [inline README.md:57]: - [ ] I congratulate you on using the Microverse template 👏 . However, there are a lot of changes that need to be applied to make it more professional. You are not supposed to delete the components of the microverse Readme template but instead you are mandated to customize it. Kindly ensure to make editions but adding your personal details and project details ☺️ 👍 .

File: README.md [hunk 2]
```diff
@@ -1,2 +1,57 @@
...
+
+### Tech Stack <a name="tech-stack"></a>
+
+> The first HTML and CSS project from Microverse
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
Comment (AlphaNtihinduka) [inline README.md:55]: - [ ] I have noticed that the MIT link is broken. The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. Kindly ensure that the MIT link is properly working and leading to the proper license ☺️ .

**Comment by abeltsew** [comment]
Hi @wayungi,

Thank you for your valuable comments, I have updated the ReadMe file as per your instructions and created a new review request from my student dashboard.

**Review by AlphaNtihinduka — CHANGES_REQUESTED** [review]
## Hi @abeltsew 🙋‍♂️

Good job working on your first contact form. You have done the design to the best expectation and incredibly well. 👏  

However, there are some issues mostly on readme.

You still need to implement few on the project but you are almost there!!! :watch: 

TO HIGHLIGHT:
- Your project is professional ✅
- Professional README file :100: 
- Your PR is professional ✅
- Well-structured files ✅

# STATUS: Required Changes :recycle: 

close

# Required changes ♻️ 
- [ ]  Please ensure that the feature branch is more descriptive. I would suggest you consider the use of `hello_microverse` or any other descriptive name to five a little overview of what the branch is all about ☺️ .  Because of this i request you to  checkout to new branch and use the correct name and mention the old PR in the new PR.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread or slack channel if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

HAPPY CODING AND GOOD LUCK :crossed_fingers: :four_leaf_clover:

**Comment by abeltsew** [comment]
Hi @AlphaNtihinduka,

Thank you for the feedback, I have renamed the feature branch to your suggested name "hello-microverse"

**Pull request closed without merging**